numpy/core/src/umath/funcs.inc.src

```diff
@@ -267,6 +267,38 @@ npy_ObjectClip(PyObject *arr, PyObject *min, PyObject *max) {
     return o;
 }
 
+static PyObject *
+npy_ObjectPopCount(PyObject *obj) {
+    PyObject *result;
+
+    /* Try to use inbuilt popcount if available */
+    {
```

Comment:
Do you prefer the additional scope, or is it just C89 compatibility (if the latter, might as well drop it)

Reply:
I actually followed Erics GCD implementation. Sure, I shall remove it.

Reply:
Fixed in 4e78f17